e2e/components/UnderlineNav.test.ts

```diff
@@ -0,0 +1,159 @@
+import {test, expect} from '@playwright/test'
+import {visit} from '../test-helpers/storybook'
+import {themes} from '../test-helpers/themes'
+import {viewports} from '../test-helpers/viewports'
+
+test.describe('UnderlineNav', () => {
+  Object.keys(viewports).forEach(key => {
+    themes.forEach(theme => {
+      test(`VRT @ ${key} (${viewports[key]}px) on ${theme} theme`, async ({page}) => {
```

Comment:
What do you think of having the test structure be something like:

- Top-level: component name
- First-level: story name
- Second-level: theme + modifiers + tags

For example:

```ts
test.describe('UnderlineNav', () => {
  test.describe('Internal Responsive Nav', () => {
    themes.forEach(theme => {
      Object.entries(viewports).forEach(([viewport, width]) => {
        test(`${theme} (${viewport}) @vrt`, async ({ page }) => {
        })
      })
    });
  });
});
```

Reply:
Ahh I guess I didn't think that we need to cover every story to VRT test 🙈  I rather chose one story to test the themes and viewports and write possible interaction tests based on the same story but I see now that is not the best approach. Thanks for mentioning that and yes your suggested structure makes perfect sense. I'll push a commit to update them